templates/template-post.html

```diff
@@ -116,14 +122,26 @@
             <article>{{ ref.item.text | safe }}</article>
           {% endif %}
           </div>
-          {% if ref.item.item_type == "image" and loop.index0 - 1 >= 0 and item.groups[1].refs[loop.index0 - 1].item.item_type == "image" and not (loop.index0 + 1 >= loop.length or item.groups[1].refs[loop.index0 + 1].item.item_type == "image") %}
-          </div>
+          <!-- item end -->
+          {% if settings.showGallery and ref.item.item_type == "image" and loop.index0 - 1 >= 0 and item.groups[1].refs[loop.index0 - 1].item.item_type == "image" and item.groups[1].refs[loop.index0 + 1].item.item_type != "image" %}
```

Comment:
this condition isn't good
`item.groups[1].refs[loop.index0 + 1].item.item_type ` it can be that the current element it is the last.

Reply:
that was planned :) `undefined != "image"` is true